src/components/Modals/AddAddress/index.tsx

```diff
@@ -129,6 +129,21 @@ export const AddAddressModal = ({ open, onClose }: AddAddressModalProps) => {
                   </MenuItem>
                 ))}
               </TextField>
+              {networkId !== undefined ? (
```

Comment:
Can't we just have this instead: 
```suggestion
              {networkId !== undefined &&
```